test/client_logger_test.rb

```diff
@@ -0,0 +1,75 @@
+require File.expand_path('../test_helper', __FILE__)
+
+class ClientLoggerTest < FaunaTest
+  def setup
+    super
+    @class_ref = client.post(:classes, name: :logging_tests)[:ref]
+  end
+
+  def test_logging
+    logged = capture_logged &:ping
+    lines = logged.split "\n"
+    read_line = lambda do
+      lines.shift
+    end
+
+    assert_equal 'Fauna GET /ping', read_line.call
+    assert_match(/^  Credentials:/, read_line.call)
+    assert_equal '  Response headers: {', read_line.call
+    # Skip through headers
+    loop do
+      line = read_line.call
+      unless line.start_with? '    '
+        assert_equal '  }', line
+        break
+      end
+    end
+    assert_equal '  Response JSON: {', read_line.call
+    assert_equal '    "resource": "Scope global is OK"', read_line.call
+    assert_equal '  }', read_line.call
+    assert_match(/^  Response \(200\): API processing \d+ms, network latency \d+ms$/, read_line.call)
+  end
+
+  def test_request_content
+    logged = capture_logged do |client|
+      client.post @class_ref, data: {}
+    end
+
+    lines = logged.split "\n"
+    read_line = lambda do
+      lines.shift
+    end
+
+    assert_equal 'Fauna POST /classes/logging_tests', read_line.call
+    assert_match(/^  Credentials:/, read_line.call)
+    assert_equal '  Request JSON: {', read_line.call
+    assert_equal '    "data": {', read_line.call
+    close_data = read_line.call
+    if close_data == '  '
+      # JRuby may add an extra empty line here.
```

Comment:
Can you update the comment to be a bit more specific with what's happening? Ie the formatting difference.